modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/resource-permissions.json

```
@@ -1,187 +1,165 @@
 [
 	{
-		"actionIds": [
-			"VIEW"
-		],
+		"actionIds": ["VIEW"],
 		"resourceName": "com.liferay.journal.model.JournalFolder",
 		"roleName": "User",
 		"scope": "1"
 	},
 	{
-		"actionIds": [
-			"SUBSCRIBE",
-			"VIEW"
-		],
+		"actionIds": ["SUBSCRIBE", "VIEW"],
 		"resourceName": "com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet",
 		"roleName": "User",
 		"scope": "1"
 	},
 	{
-		"actionIds": [
-			"VIEW"
-		],
+		"actionIds": ["VIEW"],
 		"resourceName": "com.liferay.list.type.model.ListTypeDefinition",
 		"roleName": "User",
 		"scope": "1"
 	},
 	{
-		"actionIds": [
-			"VIEW"
-		],
+		"actionIds": ["VIEW"],
 		"primKey": "[$OBJECT_DEFINITION_ID:MDFRequest$]",
 		"resourceName": "com.liferay.object.model.ObjectDefinition",
 		"roleName": "User",
 		"scope": "4"
 	},
 	{
-		"actionIds": [
-			"VIEW"
-		],
+		"actionIds": ["UPDATE", "VIEW"],
+		"primKey": 0,
```

```suggestion
		"primKey": 0,
```

This field is necessary?